sdk/schemaregistry/schema-registry-avro/README.md

```diff
@@ -88,20 +86,24 @@ const deserializedValue = await serializer.deserialize(buffer);
 
 ## Troubleshooting
 
-### Enable logs
+### Logging
 
-You can set the following environment variable to see debug logs when using this library.
+Enabling logging may help uncover useful information about failures. In order to
+see a log of HTTP requests and responses, set the `AZURE_LOG_LEVEL` environment
+variable to `info`. Alternatively, logging can be enabled at runtime by calling
+`setLogLevel` in the `@azure/logger`:
 
-- Getting debug logs from the Azure Schema Registry client library
+```javascript
+import { setLogLevel } from "@azure/logger";
```

Comment:
nit: js examples use `require` still

Reply:
Doh. This is copied from https://www.npmjs.com/package/@azure/identity#troubleshooting

Reply:
Fixed.

Reply:
/cc @jonathandturner to fix identity for us ;)